plugins/win-capture/data/locale/en-US.ini

```diff
@@ -42,12 +42,14 @@ Mode="Mode"
 
 # Generic compatibility messages
 Compatibility.GameCapture.Admin="%name% may require OBS to be run as admin to use Game Capture."
-Compatibility.GameCapture.Blocked="%name% cannot be captured via Game Capture. Use Window Capture or Display Capture instead."
+Compatibility.GameCapture.Blocked="%name% cannot be captured using Game Capture. Use Window Capture or Display Capture instead."
+Compatibility.GameCapture.Blocked.Applications="%name% applications cannot be captured using Game Capture. Use Window Capture or Display Capture instead."
 Compatibility.GameCapture.WrongGPU="If the preview is blank, make sure %name% is running on the same GPU as OBS."
 Compatibility.WindowCapture.BitBlt="%name% may not be capturable using the selected Capture Method (BitBlt)."
+Compatibility.WindowCapture.BitBlt.Applications="%name% applications may not be capturable using the selected Capture Method (BitBlt)."
 
 # Specific compatibility messages
 Compatibility.Application.CSGO="CS:GO may require the <code>-allow_third_party_software</code> launch option to use Game Capture."
-Compatibility.Application.ElectronGameCapture="Games built on Electron cannot be captured using Game Capture. Use Window Capture or Display Capture instead."
-Compatibility.Application.ElectronBitBlt="Electron-based applications may not be capturable using the selected Capture Method (BitBlt)."
+Compatibility.Application.ElectronGameCapture="Games built on Electron/Chromium cannot be captured using Game Capture. Use Window Capture or Display Capture instead."
+Compatibility.Application.ElectronBitBlt="Based on Electron/Chromium applications may not be capturable using the selected Capture Method (BitBlt)."
```

Comment:
Actually the grammar here is wrong, it should either be "Applications based on Electron/Chromium" or "Electron/Chromium-based applications".